parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -102,6 +102,10 @@
 
   public static String PARQUET_READ_PARALLELISM = "parquet.metadata.read.parallelism";
 
+  // configure if we want to use Hadoop's V2 read(bytebuffer). If true, we try to read using the
+  // new Hadoop read(ByteBuffer) api. Else, we skip.
+  public static String PARQUET_HADOOP_BYTEBUFFER_READ = "parquet.read.useByteBuffer";
```

Comment:
maybe `parquet.read.use.byte.buffer`? I'm not sure what the convention here has been in the past but i think it's been just lots of `.`s